roadmap/src/components/Roadmap/Roadmap.js

```diff
@@ -1,88 +1,149 @@
+      title: "Exploring",
+      tickets: exploringTickets,
+    },
+    {
+      title: "In Progress",
+      tickets: inProgressTickets,
+    },
+    {
+      title: "Done",
+      tickets: doneTickets,
+    },
+    {
+      title: "Leaving it for now",
+      tickets: leavingItForNowTickets,
+    },
+  ];
+
+  const handleUpvoteTicket = React.useCallback(
+    async (id) => {
+      setUpvotes((upvotes) => [...upvotes, id]);
+      localStorage.setItem(`${id}`, true);
+
+      upvoteTicket({ variables: { ticketId: id } });
+    },
+    [upvoteTicket]
+  );
+
+  const isTicketUpvoted = React.useCallback(
+    (id) => !!upvotes.find((t) => t === id) || localStorage.getItem(id),
+    [upvotes]
+  );
+
   return (
     <div className="roadmap">
-      <div className="roadmap-column">
-        <div className="roadmap-column-heading">Exploring</div>
-        <div className="roadmap-cards">
-          {exploringTickets.map((t, i) => (
-            <div key={`exploring-${i}`} className="roadmap-cards-item">
-              <div className="roadmap-cards-item-heading">{t.title}</div>
-              <div
-                className="roadmap-cards-item-content"
-                dangerouslySetInnerHTML={{ __html: t.description }}
-              />
-            </div>
-          ))}
-        </div>
-      </div>
-      <div className="roadmap-column">
-        <div className="roadmap-column-heading">In Progress</div>
-        <div className="roadmap-cards">
-          {inProgressTickets.map((t, i) => (
-            <div key={`exploring-${i}`} className="roadmap-cards-item">
-              <div className="roadmap-cards-item-heading">{t.title}</div>
-              <div
-                className="roadmap-cards-item-content"
-                dangerouslySetInnerHTML={{ __html: t.description }}
-              />
-            </div>
-          ))}
-        </div>
-      </div>
-      <div className="roadmap-column">
-        <div className="roadmap-column-heading">Done</div>
-        <div className="roadmap-cards">
-          {doneTickets.map((t, i) => (
-            <div key={`exploring-${i}`} className="roadmap-cards-item">
-              <div className="roadmap-cards-item-heading">{t.title}</div>
-              <div
-                className="roadmap-cards-item-content"
-                dangerouslySetInnerHTML={{ __html: t.description }}
-              />
-            </div>
-          ))}
-        </div>
-      </div>
-      <div className="roadmap-column">
-        <div className="roadmap-column-heading">Leaving it for now</div>
-        <div className="roadmap-cards">
-          {leavingItForNowTickets.map((t, i) => (
-            <div key={`exploring-${i}`} className="roadmap-cards-item">
-              <div className="roadmap-cards-item-heading">{t.title}</div>
-              <div
-                className="roadmap-cards-item-content"
-                dangerouslySetInnerHTML={{ __html: t.description }}
-              />
+      {loading ? (
+        <svg
```

Comment:
Import the svg file 
Ex: `import { ReactComponent as LoaderIcon } from '../icons/loader.svg'`